collectors/python.d.plugin/beanstalk/metadata.yaml

```diff
@@ -0,0 +1,207 @@
+    description: |
+      This collector works by... {[TBD](link to TBD)} ...
+  supported_platforms:
+    description: |
+      If this collector can't run on some platforms specify them here. Leave empty otherwise.
+  multi-instance: true
+  permissions:
+    description: |
+      if there are any notable permission requirements, eg running as root and so on, specify them here. Leave empty otherwise.
+setup:
+  prerequisites:
+    list:
+      - title: Prerequisite title for heading
+        text: |
+          - TBD
+          - TBD
+  behavior:
+    auto_detection:
+      description: |
+        TBD
+    limits:
+      description: |
+        TBD
+    performance_impact:
+      description: |
+        TBD
+  configuration:
+    file:
+      description: |
+        which file the user has to edit. Example: go.d/apache.conf
+    options:
+      description: |
+        TBD, use | and newline for multiline
+      folding:
+        title: Config options
+        enabled: true
+      list:
+        - name: option name
+          description: option description
+          default: a number or a string
+          required: true or false
+    examples:
+      folding:
+        title: Config
+        enabled: false
+      list:
+        - name: Basic / else describe what this example is about
+          description: |
+            A basic example configuration / describe your example
+          data:
+            jobs:
+              - name: example
+                TBD: TBD
+troubleshooting:
+  problems:
+    list:
+      - name: possible troubleshooting title
+        text: |
+          describe the issue and the troubleshooting process
+related_resources:
+  integrations:
+    list: [apache, weblog, httpcheck, etc]
+        # This text will be added in the overview section
+    overview_text: |
+      give a simple description if there are any other collectors related to this collector. Example -> To get the most out of TBD, you can do XYZ with collectors X Y and Z. Check our related resources for more information.
+    provide:
+      description: |
+        What info I can provide for other plugins that reference me? this will only be visible in their end, they will scrape this text. Example -> XYZ allows monitoring X Y Z metrics...
+alerts:
+  - name: beanstalk_server_buried_jobs
+    link: https://gihtub.com/netdata/netdata/blob/master/health/health.d/beanstalkd.conf
+    metric: beanstalk.current_jobs
+    info: number of buried jobs across all tubes. You need to manually kick them so
+      they can be processed. Presence of buried jobs in a tube does not affect new
+      jobs.
+    os: freebsd
+metrics:
+  description: some description for this section, or leave empty
+  folding:
+    title: Metrics
+    enabled: false
+  scope:
+    global:
+      name: global
+      description: TBD
+      labels: []
+      metrics:
```

Comment:
use of `metrics` here is potentially new to netdata and maybe a little inconsistent. am wondering if we have to go with contexts > dimensions like in here:

https://london.my-netdata.io/api/v2/contexts
https://london.my-netdata.io/api/v1/charts

Reply:
might just be me but i think of `beanstalk.cpu_usage.user` and `beanstalk.cpu_usage.system` as 2 metrics but this implies they are both under the `beanstalk.cpu_usage` metric

Reply:
yeah, reason it is metrics, is because we named the CSVs metrics.csv 🤔

Reply:
Same as above though, this doesn't have to be the final name in the presentation, it is just for the editor's (and maybe the CI's) eyes..

Reply:
@Ancairon : I think these are contexts in the real essence.. Good to check with @ilyam8  if we want to rename this.

Reply:
These are metrics

Reply:
I agree.. I think the users are not aware of contexts etc which is more a Netdata terminology and we should still call them metrics for a user visible yaml.